plugins/systems/sfml/SpriteComponent.hpp

```diff
@@ -0,0 +1,21 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** SpriteComponent.hpp file
+*/
+
+#include <SFML/Graphics/Sprite.hpp>
+
+// TODO choose an appropriate namespace
```

Comment:
Is this a task that is still relevant or can this comment be removed?

Reply:
This comment can definitly be removed